test/output/complex-models.json

```diff
@@ -0,0 +1,65 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v1",
+    "title": "Title was not specified"
+  },
+  "paths": {
+    "/projects/{id}.json": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Project"
+            }
+          }
+        },
+        "parameters": [{
```

Comment:
@sb8244 
You have custom formatting of ouput file so it require manual fixing each time we change something in the lib.
Please uncomment [this lines](https://github.com/apigee-127/swagger-converter/blob/master/test/test.js#L107-L108) run test and copy `complex-models.json-converted` file.

@mohsen1 What do you think about doing it similar to [this](https://github.com/lucybot/api-spec-converter/blob/master/test/convert.js#L61-L62)?